mitiq/cdr/cdr.py

```diff
@@ -74,22 +74,29 @@ def execute_with_cdr(
         scale_noise: scale_noise: Function for scaling the noise of a quantum
             circuit.
         scale_factors: Factors by which to scale the noise.
+
             - When 1.0 is the only scale factor, the method is known as CDR.
+
             - Note: When scale factors larger than 1.0 are provided, the method
-            is known as "variable-noise CDR."
+              is known as "variable-noise CDR."
+
         kwargs: Available keyword arguments are:
```

Comment:
```suggestion

```
I think it might be better to get rid of this line. With the itemized list and this line, the arg types are not formatted correctly.